### PR TITLE
Add new troubleshooting for cluster import in 2.3

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/21628

### CODE REVIEW

File: troubleshooting/trouble_already_exists.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
+
+[#troubleshopoting-cluster-with-already-exists-error]
+= Troubleshooting cluster with already exists error
+
+If you are unable to import an OCP cluster into ACM `MultiClusterHub` and receive an _Already exists_ error, follow the procedure to troubleshoot the problem.
```
Comment (swopebe) [inline troubleshooting/trouble_already_exists.adoc]: neither product name presented properly.

Comment (swopebe) [reply troubleshooting/trouble_already_exists.adoc]: error goes in `this` format.

File: troubleshooting/trouble_already_exists.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+[#symptom-cluster-already-exists-error-log]
+== Symptom: Already exists error log when importing OCP cluster
+
```
Comment (swopebe) [inline troubleshooting/trouble_already_exists.adoc:9]: product name issues throughout

see:

https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1375785039

Comment (oafischer) [reply troubleshooting/trouble_already_exists.adoc:9]: Fixed the names (I think). Thanks. Is everything OK in terms of the architecture? Talked with Chris and we were evenly split on whether this needs a new book or if we can squeeze it into [trouble_import_status.adoc](https://github.com/stolostron/rhacm-docs/blob/2.3_stage/troubleshooting/trouble_import_status.adoc) like suggested in the Bugzilla. As you can see I went with a new book.

File: troubleshooting/trouble_already_exists.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+----
+oc get all -n open-cluster-management-agent
+oc get all -n open-cluster-management-agent-addon
+----
+
+[#resolving-problem-already-exists-ocp-import]
+== Resolving the problem: Already exists when importing :ocp-short: cluster
+
+Run the following link:https://github.com/stolostron/deploy/blob/master/hack/cleanup-managed-cluster.sh[`cleanup-managed-cluster.sh`] script on the managed cluster to remove pre-existing ACM resources:
```
Comment (skeeey) [inline troubleshooting/trouble_already_exists.adoc]: @oafischer 

For this part, I think we can use below commands instead  of the script `cleanup-managed-cluster.sh`

```
oc delete namespaces open-cluster-management-agent open-cluster-management-agent-addon --wait=false
oc get crds | grep open-cluster-management.io | awk '{print $1}' | xargs oc delete crds --wait=false
oc get crds | grep open-cluster-management.io | awk '{print $1}' | xargs oc patch crds --type=merge -p '{"metadata":{"finalizers": []}}'
```

refer to https://github.com/stolostron/rhacm-docs/pull/3627/files#diff-95b70bbc800333108c43d82bb146c0d7db4936880433931163d98e89673b3fa4R79

**Review by skeeey — APPROVED** [review]

File: troubleshooting/trouble_already_exists.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
+
+[#troubleshopoting-cluster-with-already-exists-error]
+= Troubleshooting cluster with already exists error
+
+If you are unable to import an :ocp-short: cluster into :product-title-short: `MultiClusterHub` and receive an `AlreadyExists` error, follow the procedure to troubleshoot the problem.
+
+[#symptom-cluster-already-exists-error-log]
+== Symptom: Already exists error log when importing :ocp-short: cluster
+
+An error log shows up when importing an :ocp-short: cluster into :product-title-short: `MultiClusterHub`:
```
Comment (dockerymick) [inline troubleshooting/trouble_already_exists.adoc]: change `:ocp-short:` to `{ocp-short}`

same change for all of the conrefs

**Review by dockerymick — CHANGES_REQUESTED** [review]

**Review by dockerymick — APPROVED** [review]
All of the conrefs look correct to me. I recommend that you make sure the conrefs render correctly in the preview doc, before officially closing the issue. Thanks for documenting this Oliver!

**Pull request merged**